snippets/saved_replies.md

```diff
@@ -0,0 +1,103 @@
+
+To make your question, and all replies, easier to find, we suggest you move this over to our [user mailing list](https://groups.google.com/forum/#!forum/prometheus-users), which you can also search. If you prefer more interactive help, join or our [IRC channel](https://webchat.freenode.org/?channels=prometheus), #prometheus on irc.freenode.net. Please be aware that our IRC channel has no logs, is not searchable, and that people might not answer quickly if they are busy or asleep. If in doubt, you should choose the mailing list.
+
+If you think this is not purely a support question, feel free to comment in here or take the underlying issues to our [developer mailing list](https://groups.google.com/forum/#!forum/prometheus-developers).
+
+Once your questions have been answered, please add a short line pointing to relevant replies in case anyone stumbles here via a search engine in the future.
+
+All that being said, in your specific case, it might make sense to / please look at / our reasoning can be found at XXX
+
+
+# PRs
+
+
+## DCO
+
+Thanks for your PR. For legal reasons, we require that all commits are signed with a [DCO](https://developercertificate.org/) before we can merge them.
+
+This means that the last line of your commit message should read like:
+
+`Signed-Off-By: Your Name <your@email.address>`
+
+If you are using GitHub through the web interface, it's quickest to close this PR and open a new one with the appropriate line.
+
+If you are using Git on the command line, it is probably quickest to amend and force push. You can do that with
+````
+git commit --amend --signoff
+git push -f $remote $remote_branch_for_pr
+````
+
+As always, be careful when force-pushing.
+
+
+## Moratorium on integrations (SD & Alertmanager)
+
+Thanks for suggesting this change. Unfortunately, we are currently not accepting new integrations.
+
+We can only provide the stability and performance we want to provide if we can test integrations in an automated and scalable fashion. For this reason, we are suggesting people integrate with the help of our generic interfaces by either integrating natively or putting a rewriting proxy in the middle. We would be more than happy to list this on our integrations page.
+
+Even if existing integrations can not be tested in an automated fashion, we will not remove them for reasons of compability. This also means that any additions we take on, or any changes to existing integrations we make or accept, will mean maintaining and testing these until at least the next major version, realistically even beyond that.
+
+You are free to question this answer on our [developer mailing list](https://groups.google.com/forum/#!forum/prometheus-developers), but it's unlikely that you will get a different answer.
+
+
+# Issue or PR
+
+
+## Duplicate
+
+Thanks for reporting this. It seems we already have this issue tracked at XXX
+
+Thus, we are closing this one for now. If you think this was in error, don't hesitate to comment.
```

Comment:
I like to say: "If you feel this case is different, please feel free to re-open and comment on this issue."

Reply:
I don't believe they can reopen, we have to.

Reply:
Really? Then I've lied to people a ton of times :sweat_smile:, I thought the author of an issue can re-open issues.

Reply:
Just double checked, you can comment but not reopen.

Reply:
Yah, I also wrote that (as I see this often on GH), actually checked, and changed the wording. I will remove the "we" to "I", though.

Reply:
Seeing the amount of "we" I put in there, I will pull that discussion into a comment below.